### ISSUE TITLE
<a target="_blank"> within modal does not regain focus after new window is closed

### ISSUE THREAD

**Issue by dariov21** [twbs/bootstrap#17506]
How to reproduce it 

here is the jsfiddle 
https://jsfiddle.net/zzmjdf87/4/

open jsfiddle link 
click on "Lunch demo modal" button 
click in "open google" link inside modal 
this should open another tab
close this tab
result 
-> you don't have focus on "open google" link back
expected 
-> you should have the focus back in "open google" link

so -> window 7
browser -> Firefox 34.0.5

**Comment by twbs-lmvtfy** [comment]
Hi @dariov21!

You appear to have posted a live example (https://fiddle.jshell.net/zzmjdf87/2/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 20, column 7: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.5 ; saw what appears to be usage of Bootstrap 3.2.0

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by dariov21** [comment]
I have update the bootstrap version to 3.3.5 and the issue still happening

**Comment by cvrebert** [comment]
Testcase: https://jsfiddle.net/cvrebert/bg17Lx6h/
Confirmed in OS X Firefox (the `.modal` get focused instead). Does not occur in Chrome.
Confirmed in Safari, but probably not a bug there since links aren't normally focusable in Safari anyway.

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team
